### ISSUE TITLE
feature request - indicate annotation file has a subset of chroms

### ISSUE THREAD

**Issue by drmjc** [brentp/vcfanno#73]
Hi Brent,
Thanks for a really great tool. it's blazing fast!

I'm testing vcfanno to add GnomAD annotations for some WGS data. There's one GnomAD WGS VCs per-chromosome, so I have 23 separate [[annotation]] blocks in the toml config.  vcfanno isn't too happy about there not being a full set of chroms in each annotation file. Would you consider adding a 'chroms'  attribute to indicate that a given annotation file only contains a subset of chromosomes?

The warnings all relate to chromosome 1 not being found, eg:

    bix.go:200: chromosome 1 not found in gnomad.genomes.r2.0.1.sites.X.vcf.gz

cheers,
Mark

toml creation via:

```
for chr in `seq 1 22` X; do
  cat << EOF >> gnomad.toml
[[annotation]]
file="gnomad.genomes.r2.0.1.sites.${chr}.vcf.gz"
fields = ["AC_POPMAX", "AN_POPMAX", "AF_POPMAX"]
ops=["self", "self", "self"]
names=["gnomad_ac_popmax", "gnomad_an_popmax", "gnomad_ad_popmax"]

EOF
done
# overwrite any values with those from exomes.
cat << EOF >> gnomad.toml
[[annotation]]
file="gnomad.exomes.r2.0.1.sites.vcf.gz"
fields = ["AC_POPMAX", "AN_POPMAX", "AF_POPMAX"]
ops=["self", "self", "self"]
names=["gnomad_ac_popmax", "gnomad_an_popmax", "gnomad_ad_popmax"]
EOF
```

**Comment by brentp** [comment]
I think it would be preferable to make the warnings less frequent--preferably only once. I dont want to add any burden on the user to specify that. I'll have a look and see if I can only report once.

**Comment by aksenia** [comment]
Very relevant for MT dataset - there is massive amount of warnings for all other chromosomes. it would be nice to have something optional - like [filter] or [skip] for those datasets that do not have all the expected chromosomes.
